src/main/java/seedu/planitarium/parser/Parser.java

```diff
@@ -120,7 +121,10 @@ public static void isValidMoney(String amount) throws NumberFormatException {
      */
     public static void isValidUserIndex(String userIndex) throws NumberFormatException, IndexOutOfBoundsException {
         int checkIndex = Integer.parseInt(userIndex);
-        if (checkIndex < MIN_USER_INDEX || checkIndex >= PersonList.getNumberOfMembers()) {
+        if (checkIndex < MIN_USER_INDEX) {
+            throw new IndexOutOfBoundsException();
+        }
+        if (checkIndex >= PersonList.getNumberOfMembers()) {
```

Comment:
if you would like to check the index, you may use a seperate boolean function to check it as it is the requirement of coding standard.

Reply:
Thank you, will implement in future PR, will leave this PR to be focused on the bug fix